js/fileupload.js

```diff
@@ -353,13 +353,52 @@ var insertImageInTinyMCE = function(editor, image) {
  */
 if (typeof tinyMCE != 'undefined') {
     tinyMCE.PluginManager.add('glpi_upload_doc', function(editor) {
+        let last_paste_content = null;
+        const rtf_img_types = {
+            'pngblip': 'image/png',
+            'jpegblip': 'image/jpeg',
+        };
+        editor.on('paste', (e) => {
+            last_paste_content = e.clipboardData;
+        });
         editor.on('PastePreProcess', function(event) {
+            const base64_img_contents = [];
+            if (last_paste_content !== null) {
+                // Extract all RTF images and remove line breaks
+                const rtf_content = last_paste_content.getData('text/rtf');
+                const rtf_content_no_line_break = rtf_content.replace(/(\r\n|\n|\r)/gm, "");
+                const hex_binary = rtf_content_no_line_break.matchAll(/\\(pngblip|jpegblip)([a-z0-9]*)}/g);
+
+                // For each match, convert to base64
+                for (const match of hex_binary) {
+                    const img_type = match[1];
+                    const hex = match[2];
+                    const hexToBase64 = function(hexstring) {
+                        return btoa(hexstring.match(/\w{2}/g).map(function(a) {
+                            return String.fromCharCode(parseInt(a, 16));
+                        }).join(""));
+                    };
+                    base64_img_contents.push({
+                        type: rtf_img_types[img_type],
+                        content: hexToBase64(hex)
+                    });
+                }
+            }
             // Trigger upload process for each pasted image
             var fragment = $('<div></div>');
             fragment.append(event.content);
             fragment.find('img').each(function() {
                 const image = $(this);
-                const src = image.attr('src');
+                let src = image.attr('src');
+                const file_pattern = '^file://';
+                const local_file_blob_pattern = `^blob:${CFG_GLPI.url_base}/[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}`;
```

Comment:
This kind of URL is produced by `URL.createObjectURL(file)` (line 95).

I think the only way to reproduce presence of this kind of URLs is to copy the data I just pasted and paste it again in the editor. I propose to not handle this case in this PR, as it a specific case that will not be a problem anymore in another fix I am working on in order to handle #12961.

Reply:
While working on #13016, I dig a little bit on this. This is really an edge case.

As far as I understand, blob URLs are produced by TinyMCE. Technically, they are valid as long as they are used in the same browser session. So the only way to have this kind of issues is to copy an image pointing to a blob from a browser A and try to use it in a browser B.

I was able to reproduce the issue by:
 - starting to write a ticket description containing a pasted image on Chrome,
 - copying the content before it was submitted,
 - pasting it a new ticket form in Firefox.

Complexing the code to handle an unlikely case is something I do not favor.